tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_friendship.c

```diff
@@ -361,6 +361,20 @@ static void test_friend_group(void)
 	PASS();
 }
 
+/* LPN disable friendship dummy
+ *
+ * Dummy friendship node for test_lpn_disable
+ */
+static void test_friend_lpn_disable_dmy(void)
+{
+	/* Wait for LPN node to toggle lpn_set */
+	k_sleep(K_SECONDS(1));
+
+	bt_mesh_test_setup();
+	bt_mesh_friend_set(BT_MESH_FEATURE_ENABLED);
+	PASS();
```

Comment:
Should verify that no friendship is actually established, just like the LPN does.